### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_Deploy Web App_Keyboard: When there is no data present, focus is lost after tabbing from ‘Resource Group’ section to ‘App Settings’

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3340]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.4.3; #Keyboard; #C+E-H; #Java tooling for IntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS2.4.3

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace settings/preferences popup
3) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
4) Tab till File-->New-->Project--->Dynamic Web Project
5) Within Eclipse's Project Explorer view, expand Newly created dynamic web project. Right-click WebContent, click New-->JSP File
6) Tab till Next button and press enter
7) Select New JSP File (html) and press Finish
8) Open JSP file and write any code in the editor
9) In Project Explorer view, right-click your project, choose Azure-->Publish as Azure Web App
10) Tab till Create button and press enter
11) Tab till ‘App Settings’
12) Verify When there is no data present, focus is lost after tabbing from ‘Resource Group’ section to ‘App Settings’

**Actual Result:** 
When there is no data present, focus is lost after tabbing from ‘Resource Group’ section to ‘App Settings’

**Expected Result:** 
Focus should go to the next interactive control but should not get lost in App settings

**User Impact:** 
Keyboard users will find it confusing about navigating in the popup if there is any focus loss

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7D&cid=23477c83-2764-40ea-8f41-68c1a61cd871

[Bug_18_Focus lost on App settings.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451299/Bug_18_Focus.lost.on.App.settings.pptx)

**Comment by Flanker32** [comment]
@Tanushagottipati Lost focus when no data exist is a common behavior of Eclipse Table, which is out of the scope of our toolkits, here is the gif of Eclipse Preferences table behavior(Preferences->Run/Debug->TCP/IP Monitor)

![3340](https://user-images.githubusercontent.com/12445236/64516242-b922f180-d320-11e9-9b78-5dde16314f80.gif)

**Comment by Flanker32** [comment]
Importance of experience: Little, it could be better to use if we could skip the empty table and move to create directly.
Quality of experience: Little, users may get confused when there is focus lose, but they could know where they are by tab again which will make the focus move to create new button
Alternate solutions: No alternate technologies or options can be leveraged.
Evidence: All evidence is provided in this issue.
AAT Link

**Comment by Flanker32** [comment]
@Tanushagottipati This is an external issue of Eclipse and we have tracked it in AAT, please help us verify it, thanks!

**Comment by Tanushagottipati** [comment]
External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=550893

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749747
